editor/export/project_export.cpp

```diff
@@ -593,6 +594,11 @@ void ProjectExportDialog::_enc_pck_changed(bool p_pressed) {
 	enc_in_filters->set_editable(p_pressed);
 	enc_ex_filters->set_editable(p_pressed);
 	script_key->set_editable(p_pressed);
+	show_script_key->set_disabled(!p_pressed);
+	if(!p_pressed){
```

Comment:
```suggestion
	if (!p_pressed) {
```
You could use clang format to avoid problems with code style :) https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html#installation

Reply:
Thanks for the tip! :)
I looked around, and there is a default shortcut in vs-code to automatically format to clang
I also added pre-commit 😁 and did the fix, so it should be fine now